src/test/java/core/basesyntax/handler/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.handler;
+
+import core.basesyntax.db.Storage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class BalanceOperationHandlerTest {
+    private static BalanceOperationHandler balanceOperationHandler;
+
+    @BeforeAll
+    static void beforeAll() {
+        balanceOperationHandler = new BalanceOperationHandler();
+    }
+
+    @Test
+    void handle_validData_ok() {
+        boolean expectedResult = balanceOperationHandler.handle("apple", 20);
```

Comment:
You can use constant for "apple" in this and other hanlers tests

Reply:
Double "apple" in this file.